scanner/src/test/kotlin/scanners/scancode/ScanCodeResultParserTest.kt

```diff
@@ -282,7 +295,7 @@ class ScanCodeResultParserTest : WordSpec({
             val result = readJsonFile(resultFile)
 
             val actualFindings = generateSummary(Instant.now(), Instant.now(), SpdxConstants.NONE, result)
-                    .copyrightFindings
+                .copyrightFindings
```

Comment:
There's again this unrelated formatting change.

Reply:
This time it is intended, see the first commit.